lib/train/transports/local.rb

```diff
@@ -12,14 +12,14 @@ class Local < Train.plugin(1)
 
     include_options Train::Extras::CommandWrapper
 
-    autoload :File, 'train/transports/local_file'
-    autoload :OS,   'train/transports/local_os'
-
     def connection(_ = nil)
       @connection ||= Connection.new(@options)
     end
 
     class Connection < BaseConnection
+      require 'train/transports/local_file'
```

Comment:
Looks like the require had to be here as `local_file` starts out by declaring a class underneath `Connection`:

    class Train::Transports::Local::Connection
      class File < LinuxFile
        ...